earth_enterprise/src/server/geselectpublishroot

```diff
@@ -15,12 +15,31 @@
 # limitations under the License.
 #
 
+use strict;
+use Getopt::Long qw(:config no_ignore_case no_auto_abbrev);
+use FindBin qw($Script);
 
-# make sure we're running as root
-die "You must run as root\n" unless $> == 0;
+my $cmd_args_good = 0;
+my $help = 0;
+my $no_prompt = 1;
+
+sub usage {
+  warn "usage: $Script [-h] <publish_root_dir>\n";
+  warn "   Update the GE http server to point to the given\n";
+  warn "   published databases directory\n";
+  warn "Valid options are:\n";
+  warn "  -h, --help             Display this message\n";
 
-if ($#ARGV < 0) {
-  die "Usage: geselectpublishroot <publish_root_dir>\n";
+  exit(1);
+}
+
+# The --noprompt option is always implied due to requiring the dir arg on the
+# command line. Nothing different has to happen if specified. It is here for
+# consistency to avoid errors/confusion if it is specified.
```

Comment:
Sure, I have no problem removing it. I had noticed on the [command reference page](https://github.com/google/earthenterprise/blob/master/docs/geedocs/5.3.0/answer/3481558.html) that it did have a --noprompt option listed, and so that is why I had added it in the script. Maybe I should also just remove it from the documentation. It appears to have never been in the script before.

Reply:
Well! Good looking at the documentation. 👍 🙂   I actually hadn't seen that there- this makes more sense to me now.

As you mentioned, since the option was never in the script, the documentation was in error, and might lead to confusion as it is. Please go ahead and amend the documentation.